.orchestra/ci/install-system-dependencies.sh

```diff
@@ -103,7 +103,7 @@ PACKAGES+=(ripgrep)
 
 apt-get -qq install --no-install-recommends --yes "${PACKAGES[@]}"
 
-if ! which git-lfs &> /dev/null; then
+if ! command -v git-lfs &> /dev/null; then
```

Comment:
Why?

Reply:
`command -v` is:
* POSIX compatible
* a builtin of bash

Also there are some other (minor) subtle things about `which`, see [here](https://stackoverflow.com/a/677212)

Reply:
ACK. IMO none of the arguments are particularly compelling here, but it's "better" in several ways.
Please adapt all the other uses of `which' in orchestra (or if too troublesome, leave `which`).

Reply:
Dropped all uses of `which`